src/addBusinessDays/index.ts

```diff
@@ -118,29 +133,31 @@ export default function addBusinessDays(
           return businessDaysDelta
       }
     }, 0) - accountedForExceptions
+
   // Add or subtract days until we have applied all our exceptions
   while (dayChangesDueToExceptions !== 0) {
     const deltaSign = dayChangesDueToExceptions < 0 ? -1 : 1
-    if (businessDays.includes(date.getDay())) {
+    if (isWorkingDay(newDate)) {
```

Comment:
this needed to check for a working day instead of a business day, to include or exclude the exceptions correctly